### PR TITLE
Added several includes throughout Guide.

### PR DESCRIPTION
**Pull request by Bosch-0**
Following on from my other PR https://github.com/BitcoinDesign/Guide/pull/264 I added several includes that were not applied to their relevant content throughout the Guide. Mostly Dos and Don'ts and products in private key chapter.

### CODE REVIEW

File: guide/contribute/content-guidelines.md [hunk 1]
```diff
@@ -33,13 +33,17 @@ This guide is crafted for anyone interested in building an effective Bitcoin app
 
 Address the reader directly. Whenever possible, try using active instead of passive voice. Make the reader a part the conversation by using second-person pronouns like "you, your and yours". Do not write in first-person and avoid giving predictions and personal opinions.
 
-##### Do
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (GBKS) [inline guide/contribute/content-guidelines.md:36]: You can also consider definition lists for these ([see here](https://bitcoin.design/guide/contribute/formatting/#definition-list)). Those are great for a before/after representation of text.

File: guide/contribute/review.md [hunk 2]
```diff
@@ -81,16 +81,20 @@ When you're done with leaving individual feedback, you can submit it. Click on t
 
 #### Best practices when reviewing pull requests
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
 
-- Leave a suggestion instead of general feedback whenever possible, especially for typo and grammar-related feedback
```
Comment (GBKS) [inline guide/contribute/review.md:86]: I think there's a mix-up here. Some text was removed and other text is duplicated.

File: guide/foundations/principles.md [hunk 3]
```diff
@@ -172,11 +207,17 @@ The Bitcoin network doesn’t need to know your name for you to use it. Strive t
 
 Unlike traditional banking systems, the Bitcoin economy does not require new users to seek permission from anyone. Bitcoin has no central point of control. No one person or entity is in charge. Connecting to any [node]({{ '/guide/glossary/#node' | relative_url }}) on the network gives you the same rights and responsibilities, ensuring no single point of failure.
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
+
 - Design products that encourage people to run a full Bitcoin node
-- Alternatively, use a light client with the [p2p network]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki'}}) using [compact block filters]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki'}})
+- Alternatively, use a light client with the p2p network using compact block filters
```
Comment (GBKS) [inline guide/foundations/principles.md:213]: Why remove the links?

File: guide/private-key-management/multi-user-schemes.md [hunk 4]
```diff
@@ -52,38 +52,41 @@ Just like with personal schemes, external signing devices can be used to hold on
...
+{% include fact/cons.html %}
+
 - Has significant complexity and op-sec burden for multiple private keys, each of which might need a good manual backup scheme
 
+{% include fact/close.html %}
+
 ### Best practice
 
-**When to use**
+#### When to use
```
Comment (GBKS) [inline guide/private-key-management/multi-user-schemes.md:71]: The fact component was designed for specifically this section ([see here](https://www.figma.com/file/K5ahSqVzoMVBuhT5hAVRJr/bitcoin.design?node-id=850%3A6300)), so I'd also switch "When to use", "When not to use", "Variations", and "Do’s" to the fact component. Creates a consistent appearance for this loose groups of bullet categories.

File: guide/private-key-management/multi-user-schemes.md [hunk 5]
```diff
@@ -52,38 +52,41 @@ Just like with personal schemes, external signing devices can be used to hold on
...
-Few tailor-made products exist for shared wallets, but any wallet application that supports multi-key setups can be used to initiate a shared wallet.
-- [Electrum](https://electrum.org){:target="_blank"}
-- [Bluewallet](https://bluewallet.io){:target="_blank"}
-- [Specter](https://specter.solutions){:target="_blank"}
-- [Armory](https://btcarmory.com){:target="_blank"}
-- [Guarda](https://guarda.com){:target="_blank"}
-- [Revault](https://revault.dev){:target="_blank"} - in development
+{% include fact/open.html color="blue" label="Products that use this scheme" icon="rectangle" %}
+
+[Electrum](https://electrum.org){:target="_blank"}, [Bluewallet](https://bluewallet.io){:target="_blank"}, [Specter](https://specter.solutions){:target="_blank"}, [Armory](https://btcarmory.com){:target="_blank"},[Nunchuk](https://nunchuk.io/){:target="_blank"}, [Sparrow](https://sparrowwallet.com/){:target="_blank"}, [Guarda](https://guarda.com){:target="_blank"}, [Revault](https://revault.dev){:target="_blank"}.
```
Comment (GBKS) [inline guide/private-key-management/multi-user-schemes.md:87]:
```suggestion
[Electrum](https://electrum.org), [Bluewallet](https://bluewallet.io), [Specter](https://specter.solutions), [Armory](https://btcarmory.com),[Nunchuk](https://nunchuk.io/), [Sparrow](https://sparrowwallet.com/), [Guarda](https://guarda.com), [Revault](https://revault.dev).
```
We have a plug-in now that automatically adds "target=blank" to external links.

File: guide/contribute/content-guidelines.md (hunk 1 above)
Comment (Bosch-0) [reply guide/contribute/content-guidelines.md:36]: Will play around with that and see how it looks :) cheers!

File: guide/contribute/review.md (hunk 2 above)
Comment (Bosch-0) [reply guide/contribute/review.md:86]: Yep I did make a mistake here, will change it today. Don't do PRs half asleep

File: guide/foundations/principles.md (hunk 3 above)
Comment (Bosch-0) [reply guide/foundations/principles.md:213]: I copied the text from the site - long story, was not intentional, will amend today :)

File: guide/private-key-management/multi-user-schemes.md (hunk 4 above)
Comment (Bosch-0) [reply guide/private-key-management/multi-user-schemes.md:71]: I'll re-do this page applying the proper include, missed that

File: guide/private-key-management/multi-user-schemes.md (hunk 5 above)
Comment (Bosch-0) [reply guide/private-key-management/multi-user-schemes.md:87]: They were just copied and pasted from the old content - will update this today

**Review by Bosch-0 — COMMENTED** [review]
Updated PR based on feedback:

- Include fact includes throught private key management section
- Fixed a few small errors with original PR

File: guide/contribute/illustration-guidelines.md [hunk 6]
```diff
@@ -66,13 +66,17 @@ Images for this page are located in:
 
 Keep your file names in lowercase and avoid any special characters (including spaces). Keep them short and descriptive.
 
-**Don't**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (johnsBeharry) [inline guide/contribute/illustration-guidelines.md:69]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```

File: guide/contribute/illustration-guidelines.md [hunk 7]
```diff
@@ -66,13 +66,17 @@ Images for this page are located in:
...
 
-`My übercool image.PNG`
+My übercool image.png
 
-**Do**
+{% include tip/close.html %}
 
-`transaction-confirmation.png`
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (johnsBeharry) [inline guide/contribute/illustration-guidelines.md:75]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do this" %}
```

**Review by johnsBeharry — CHANGES_REQUESTED** [review]
order of these is wrong.

**Comment by danielnordh** [comment]
Sorry for being late to comment on this PR.

I prefer sticking with **Do** and **Don't** for the words (visually I'm OK with the includes).
Do's and Don'ts is a well known concept and 'Do this'/'Don't do this' feels stiffer.

File: guide/contribute/content-guidelines.md (hunk 1 above)
Comment (danielnordh) [inline guide/contribute/content-guidelines.md:36]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/contribute/content-guidelines.md [hunk 8]
```diff
@@ -33,13 +33,17 @@ This guide is crafted for anyone interested in building an effective Bitcoin app
...
 
-*Your wallet balance is 1 bitcoin (BTC). If you have recently mined a block, this could be a fresh whole bitcoin. However, it is most likely going to be an accumulation of UTXOs (change) from your previous bitcoin transactions.*
+Your wallet balance is 1 bitcoin (BTC). If you have recently mined a block, this could be a fresh whole bitcoin. However, it is most likely going to be an accumulation of UTXOs (change) from your previous bitcoin transactions.
 
-##### Don't
+{% include tip/close.html %}
 
-*My wallet balance is 1 bitcoin (BTC). If I have recently mined a block, this could be a fresh whole bitcoin. However, it is most likely going to be an accumulation of UTXOs (change) from previous bitcoin transactions.*
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/contribute/content-guidelines.md:42]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/contribute/content-guidelines.md [hunk 9]
```diff
@@ -49,13 +53,17 @@ Bitcoin is a global currency. Not everyone using this guide is a native English
 
 Focus on information relevant to the reader. Use direct, clear, concise sentences that are easy to understand. Try to reduce the word count to just the right brevity without being obscure.
 
-##### Do
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/contribute/content-guidelines.md:56]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/contribute/content-guidelines.md [hunk 10]
```diff
@@ -49,13 +53,17 @@ Bitcoin is a global currency. Not everyone using this guide is a native English
...
+
+First the application generates a wallet with a keypair. Then, the key is encrypted and backed up to a location convenient and safe for the user, yet hard to for a malicious third-party to gain access to.
+
+{% include tip/close.html %}
 
-*First the application generates a wallet with a keypair. Then, the key is encrypted and backed up to a location convenient and safe for the user, yet hard to for a malicious third-party to gain access to.*
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/contribute/content-guidelines.md:62]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/contribute/illustration-guidelines.md (hunk 6 above)
Comment (danielnordh) [inline guide/contribute/illustration-guidelines.md:69]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/contribute/illustration-guidelines.md [hunk 11]
```diff
@@ -66,13 +66,17 @@ Images for this page are located in:
...
+{% include tip/open.html color="green" icon="check" label="Do this" %}
 
-`My übercool image.PNG`
+transaction-confirmation.png
 
-**Do**
+{% include tip/close.html %}
 
-`transaction-confirmation.png`
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/contribute/illustration-guidelines.md:75]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/contribute/propose-a-change.md [hunk 12]
```diff
@@ -153,16 +153,22 @@ After changes have been published on your copy, it's time to announce them in th
...
+
+- If your pull request is related to an issue, reference it. For example, “Fixes issue #32, “Closes #32” or “Related to #32”
 - Summarize changes you did and provide reasoning if necessary
 - Keep pull requests related to a specific problem. Pull requests with lots of changes and different issues are harder to review
-- The title of the pull request should concisely explain what it's trying to solve
+- The title of the pull request should concisely explain what it’s trying to solve
+
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/contribute/propose-a-change.md:165]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/contribute/review.md [hunk 13]
```diff
@@ -81,16 +81,21 @@ When you're done with leaving individual feedback, you can submit it. Click on t
 
 #### Best practices when reviewing pull requests
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/contribute/review.md:84]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/contribute/review.md [hunk 14]
```diff
@@ -81,16 +81,21 @@ When you're done with leaving individual feedback, you can submit it. Click on t
...
 
 - Leave a suggestion instead of general feedback whenever possible, especially for typo and grammar-related feedback
 - Be humble and constructive when reviewing
 - Be understanding and welcoming towards new contributors
 - If something is not clear enough, feel free to ask questions to the pull request author
 
-**Don't**
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/contribute/review.md:93]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/foundations/principles.md [hunk 15]
```diff
@@ -51,13 +51,19 @@ These are principles we in the Bitcoin Design Community identified and stand beh
 
 Our existing mental models of access to digital services are usernames and passwords controlled by a company with custody of your funds and data. With everyone having direct access to the Bitcoin network, we no longer need to design products that require people to delegate control of their funds to middlemen. While it comes with greater responsibility, self-custody enables the open financial system of peer-to-peer transactions.
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:54]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/foundations/principles.md [hunk 16]
```diff
@@ -51,13 +51,19 @@ These are principles we in the Bitcoin Design Community identified and stand beh
...
+
 - Let users control their bitcoin and private keys directly
 - Create an easy path to self-custody for Bitcoin beginners
 
-**Don't**
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:61]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/foundations/principles.md [hunk 17]
```diff
@@ -73,9 +79,9 @@ Education and awareness are a big part of security, as they can protect users fr
 
 Security can also be a feeling. A polished, good-looking, easy-to-use product that transparently communicates how it works can help users feel more secure– especially when compared to another product with the same security measures - but lacks these qualities.
 
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:82]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/foundations/principles.md [hunk 18]
```diff
@@ -84,10 +90,15 @@ Security can also be a feeling. A polished, good-looking, easy-to-use product th
 - Continuously educate users on best practice and risks
 - Reduce attack surfaces by minimizing use of external code dependencies
 
-**Don't**
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:95]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/foundations/principles.md [hunk 19]
```diff
@@ -99,19 +110,25 @@ There are no background checks, credit checks, or gatekeepers to Bitcoin. A Keny
 
 While Bitcoin is already used by a large number of people, it pales in comparison with the many more that are likely to use it in the future. We need to design products that are prepared for people unfamiliar with Bitcoin. This means using plain and familiar language, explaining things in the context where they are needed, not overwhelming people with technical detail, and more.
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:113]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/foundations/principles.md [hunk 20]
```diff
@@ -99,19 +110,25 @@ There are no background checks, credit checks, or gatekeepers to Bitcoin. A Keny
...
 - Design Bitcoin products that are usable by the widest range of people possible
 - Use plain language that people new to Bitcoin can understand regardless of prior knowledge
 - Localize your product and make it multilingual
 - Educate in place, when people are presented with a new concept
 - Treat users who rely on assistive technologies as first-class citizens
 
-**Don't**
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:124]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/foundations/principles.md [hunk 21]
```diff
@@ -120,16 +137,22 @@ While Bitcoin is already used by a large number of people, it pales in compariso
 
 Bitcoin is an open-source protocol, operating in a decentralized manner. This has led to a number of standards being developed to ensure compatibility between products. It should be easy to switch and move your Bitcoin wallet to a different application, should you wish. Ensuring that your product supports as many of these standards as possible is best practice and builds trust. More on [wallet interoperability]({{ '/guide/foundations/wallet-interoperability/' | relative_url }}).
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:140]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/foundations/principles.md [hunk 22]
```diff
@@ -120,16 +137,22 @@ While Bitcoin is already used by a large number of people, it pales in compariso
...
+
 - Support import and export of wallets
 - Support as many relevant BIPs as possible
 - Be transparent with which ones you do and don’t support
 - Maximize backwards compatibility
 
-**Don't**
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:149]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/foundations/principles.md [hunk 23]
```diff
@@ -138,14 +161,20 @@ Bitcoin is an open-source protocol, operating in a decentralized manner. This ha
 
 While an open and decentralized financial system that users can connect with directly is great, it puts a burden on them to choose a product that they trust and like to use. We can make this easier by freely sharing information about how our products work and what technologies they use/rely on. By open-sourcing your code, you can let people verify that your claims are true, ultimately building more trust with your users. It is important to be transparent with users about the risks that come with self-custodying funds. Be sure to educate about scenarios where they may risk losing access to their funds along with best practices for avoiding this.
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:164]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/foundations/principles.md [hunk 24]
```diff
@@ -138,14 +161,20 @@ Bitcoin is an open-source protocol, operating in a decentralized manner. This ha
...
+
 - Be open and transparent with how your product works
 - Let people verify your claims by open-sourcing your code when possible
 - Explain what risks the user is taking on, and how best to mitigate them
 
-**Don't**
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:172]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/foundations/principles.md [hunk 25]
```diff
@@ -156,14 +185,20 @@ A common misconception of Bitcoin is that it provides complete anonymity and pri
 
 The Bitcoin network doesn’t need to know your name for you to use it. Strive to collect as little personal information as possible about your users. When absolutely required to provide the product services, collect only the bare minimum and consider if and when this can be discarded when no longer necessary. If you collect personal information, be transparent about why and how you will use and store it.
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:188]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/foundations/principles.md [hunk 26]
```diff
@@ -156,14 +185,20 @@ A common misconception of Bitcoin is that it provides complete anonymity and pri
...
+
 - Minimize the personal information you collect
 - Avoid address reuse
 - Embrace privacy-preserving options when relevant (running a full node, compact block filters, Tor, Lightning Network, coin selection, schnorr signatures, payjoin, coinswap, etc.)
 
-**Don't**
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:196]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

File: guide/foundations/principles.md [hunk 27]
```diff
@@ -172,11 +207,17 @@ The Bitcoin network doesn’t need to know your name for you to use it. Strive t
 
 Unlike traditional banking systems, the Bitcoin economy does not require new users to seek permission from anyone. Bitcoin has no central point of control. No one person or entity is in charge. Connecting to any [node]({{ '/guide/glossary/#node' | relative_url }}) on the network gives you the same rights and responsibilities, ensuring no single point of failure.
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:210]:
```suggestion
{% include tip/open.html color="green" icon="check" label="Do" %}
```

File: guide/foundations/principles.md [hunk 28]
```diff
@@ -172,11 +207,17 @@ The Bitcoin network doesn’t need to know your name for you to use it. Strive t
...
+
 - Design products that encourage people to run a full Bitcoin node
 - Alternatively, use a light client with the [p2p network]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0157.mediawiki'}}) using [compact block filters]({{ 'https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki'}})
 - Offer the user choice of what node and other external services to connect to
 
-**Don't**
+{% include tip/close.html %}
+
+{% include tip/open.html color="red" icon="forbid" label="Don't do this" %}
```
Comment (danielnordh) [inline guide/foundations/principles.md:218]:
```suggestion
{% include tip/open.html color="red" icon="forbid" label="Don't" %}
```

**Comment by Bosch-0** [comment]
Closing this for now as there isn't a consensus on using these includes or using do's/don'ts or do this/don't do this

**Pull request closed without merging**